tests/notify.rs

```diff
@@ -103,6 +148,11 @@ fn validate_watch_dir<F, W>(ctor: F) where
   let mut w = ctor(tx).unwrap();
 
   w.watch(dir.path()).unwrap();
+  // Windows needs some time for thread spinup before we start creating files.
```

Comment:
The watch server is in a separate thread, so it gets the message async via a channel.  Were you thinking of something like a busy wait here for a confirmation from the server?  In my other branch I've added a "MetaEvent" capability to the windows watcher.  It was intended for tests but could be used to implement this wait.  https://github.com/jmquigs/rsnotify/blob/7dc1f555ff065154eb9e57134631e52bb958c6dc/src/windows.rs#L41